solutions/devsprint-michael-douglas-2/DeliveryApp/Screens/Components/LoadingView.swift

```diff
@@ -0,0 +1,101 @@
+//
+//  LoadingView.swift
+//  DeliveryApp
+//
+//  Created by Hyago Henrique on 21/02/22.
+//
+
+import UIKit
+
+final class LoadingView: UIView {
+    
+    private let isTesting: Bool
```

Comment:
Ao invés de utilizar `isTesting`, modifique o nome para `shouldAnimate`. Assim fica como se fosse uma dependência injetável normal do componente.
```suggestion
    private let shouldAnimate: Bool
```